code/modules/mining/lavaland/ash_flora.dm

```diff
@@ -251,7 +251,6 @@
 	growthstages = 3
 	rarity = 20
 	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1)
-	resistance_flags = FIRE_PROOF
```

Comment:
I removed this line because ash flora gain fireproof by the fire resistance gene on spawn.

Reply:
👍